goldstone/core/models.py

```diff
@@ -2696,6 +2701,69 @@ def build_alert_template(self, hits):
         return msg_params_dict
 
 
+class AlertSearchSQLQuery(AlertSearch):
+
+    db_table = models.CharField(max_length=64,
```

Comment:
index_prefix matches more closely with the column of the table we want to search, rather than the table-name. 

db_table matches more closely with the source of our search : an elastic-search index tree or a SQL database. 

So the mapping that I'm thinking of looks like this : 

search_source = ['es_index_tree', 'sql-table-name'] 
search_field (or index-prefix) = ['es_index-name', 'sql_column_name']